packages/gridjs/src/view/plugin/search/search.tsx

```diff
@@ -1,18 +1,19 @@
-import { h } from 'preact';
+import { ComponentChild, h } from 'preact';
 import GlobalSearchFilter from '../../../pipeline/filter/globalSearch';
 import { classJoin, className } from '../../../util/className';
 import { SearchStore, SearchStoreState } from './store';
 import { SearchActions } from './actions';
 import ServerGlobalSearchFilter from '../../../pipeline/filter/serverGlobalSearch';
 import { debounce } from '../../../util/debounce';
-import { TCell } from '../../../types';
+import { OneDArray, TCell, TColumn } from '../../../types';
 import { PluginBaseComponent, PluginBaseProps } from '../../../plugin';
 
 export interface SearchConfig {
   keyword?: string;
   enabled?: boolean;
   debounceTimeout?: number;
   selector?: (cell: TCell, rowIndex: number, cellIndex: number) => string;
+  columns?: OneDArray<TColumn | string | ComponentChild>;
```

Comment:
I don't think you need to pass the columns to this component. Columns should be available in `this.config.header...`

Reply:
I did the change you asked. Can you check again please.

Reply:
Hi @afshinm , can you recheck the pr please?